Documentation/Changelog/9.0/Breaking-452-RemoveHooks.rst

```diff
@@ -0,0 +1,29 @@
+.. include:: ../../Includes.txt
+
+=============================
+Breaking: #452 - Remove Hooks
+=============================
+
+See :issue:`452`
+
+Description
+===========
+
+Existing Hooks have been removed. They were replaced by EventListeners.
+The extension offered following hooks:
+
+* `showCartActionAfterCartWasLoaded` in `\Extcode\Cart\Controller\Cart\CartController`
+
+Affected Installations
+======================
+
+All installations that used the hooks to programmatically adjust the behavior
+of the extension are affected.
+
+Migration
+=========
+
+* `showCartActionAfterCartWasLoaded` in `\Extcode\Cart\Controller\Cart\CartController`
+  now needs to listen to the event `\Extcode\Cart\Event\Cart\BeforeShowCartEvent`
+
```

Comment:
As the old hook allowed to change the addresses it might be good to name the newly introduced events as well.

Reply:
I added both classes because, as you said, they could also be changed by the SignalSlot before. I was of the opinion that the change of both properties could also be done with the events in the session handler with the AfterRestoreAddressEvent and BeforeWriteAddressEvent. Maybe I will work on this later for TYPO3 v13.